cypress/e2e/onboarding.cy.ts

```diff
@@ -0,0 +1,42 @@
+import { urls } from 'scenes/urls'
+import { decideResponse } from '../fixtures/api/decide'
+
+describe('Onboarding', () => {
+    beforeEach(() => {
+        cy.intercept('https://app.posthog.com/decide/*', (req) =>
+            req.reply(
+                decideResponse({
+                    'product-intro-pages': 'test',
+                })
+            )
+        )
+    })
+
+    it('Navigate between /products to /onboarding to a product intro page', () => {
+        cy.visit('/products')
+
+        // Get started on product analytics onboarding
+        cy.get('[data-attr=product_analytics-get-started-button]').click()
+
+        // Confirm product intro is not included as the first step in the upper right breadcrumbs
+        cy.get('[data-attr=onboarding-breadcrumbs] > :first-child > * span').should('not.contain', 'Product Intro')
+
+        // Navigate to the product intro page by clicking the left side bar
+        cy.get('[data-attr=menu-item-replay').click()
+
+        // Confirm we're on the product_intro page
```

Comment:
This step will fail without the changes in this PR because we'll end up stuck on the install step.